src/rest/Facet/FacetSearchRequest.ts

```diff
@@ -41,4 +64,30 @@ export interface IFacetSearchRequest {
    * See the [query]{@link IQuery} documentation.
    */
   searchContext?: IQuery;
+  /**
+   * The character to use to split field values into a hierarchical sequence.
+   *
+   * **Example:**
+   *
+   * For a multi-value field containing the following values:
+   * ```
+   * c; c>folder2; c>folder2>folder3;
+   * ```
+   * The delimiting character is `>`.
+   *
+   * For a hierarchical field containing the following values:
```

Comment:
Are hierarchical fields still a thing? I mean, as a field type. I'm not sure anymore 🤔

Reply:
Last time I checked a few days ago, it was still available in Search API. In fact, one of the sources I use to test has a hierarchical field.

Reply:
It's quite new actually